self_supervised_pretraining/swinunetr_pretrained/README.md

```diff
@@ -0,0 +1,78 @@
+# Pre-trained SwinUNETR Backbone on ~50,000 3D Volumes
+
+This tutorial is an example of how to use the latest set of SwinUNETR pre-trained weights. The pre-training was done using multiple public datasets with over ~58,000 3D CT and MR volumes.
+
+Benchmark on BTCV with using the pre-trained weights as compared to random initialization shows promising gains in terms of accuracy/performance.
+
+### Steps to run the tutorial
+1. Download the dataset [BTCV](https://www.synapse.org/#!Synapse:syn3193805/wiki/217789) (More detail is provided in the Data section)
+2. Download the pretrained SwinUNETR weights on the 50,000 3D volumes from the following link. More information about the dataset and training is provided below
+3. Modify the paths for pretrained_checkpoint, data_root, json_path & logdir in ssl_train.ipynb
+4. Run the 'swinunetr_finetune.ipynb'
+
+### 1. Data
+Pre-training Dataset: Datasets from multiple sources has been used. Below is a list of all the different datasets that were utilized for training the SwinUNETR network architecture:
+
+| Dataset Name            | Source          | Body Region            |
+|-------------------------|-----------------|------------------------|
+| Aomic-1000              | Open Neuro      |                        |
```

Comment:
Missing body regions

Reply:
This is fixed, it will be number of samples instead.